pkg/server/plugin/upstreamauthority/gcpcas/gcpcas_test.go

```diff
@@ -0,0 +1,175 @@
+package gcpcas
+
+import (
+	"context"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/spiffe/spire/pkg/common/pemutil"
+	commonutil "github.com/spiffe/spire/pkg/common/util"
+	"github.com/spiffe/spire/pkg/common/x509util"
+	"github.com/spiffe/spire/pkg/server/plugin/upstreamauthority"
+	"github.com/spiffe/spire/proto/spire/common/plugin"
+	"github.com/spiffe/spire/test/spiretest"
+	"github.com/stretchr/testify/require"
+	privatecapb "google.golang.org/genproto/googleapis/cloud/security/privateca/v1beta1"
+)
+
+func generateCert(cn string, issuer *x509.Certificate, issuerKey crypto.PrivateKey) (*x509.Certificate, crypto.PrivateKey, error) {
+	priv, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
```

Comment:
why not to use keys we have on fixture? https://github.com/spiffe/spire/blob/master/test/testkey/new.go#L41

Reply:
I am using it now